Makefile

```diff
@@ -15,7 +15,7 @@
 SHELL := /bin/bash
 
 .PHONY: setup
-setup: kubeconfig
+setup: platform/kube/config
```

Comment:
@vadimeisenbergibm can you please explain more for why does this fix the issue? I have a try and this works.

Reply:
Otherwise, running `make setup` twice fails - the second `ln -s` fails for me, on ubuntu.
My change also will allow the user to set platform/kube/config differently, and `make setup` will not reset it.

Reply:
@gyliu513 regarding why it fixes the issue - make will check if `platform/kube/config` exists, and if it does, make will not perform `ln -s`.

Reply:
Got it, thanks @vadimeisenbergibm

Reply:
@vadimeisenbergibm you may also want to update the document a bit here:

doc/build.md:To avoid a `missing input file '//platform/kube:config'` error you need to `ln -s ~/.kube/config platform/kube/` or `make kubeconfig`, see [testing](testing.md) for details/why.

Reply:
@gyliu513 yep, I want to rewrite this document to make it more clear.

Reply:
where shd make kubeconfig command be run? getting  missing input file '//platform/kube:config'   error

Reply:
Regardless of the above change the code fails. From where is the file ~/.kube/config  generated?Do we need to install kubernetes for this?